travel_platform/travel_app/urls.py

```diff
@@ -4,4 +4,5 @@
 
 urlpatterns = [
     url(r'^$', views.index, name='index'),
+    url(r'^.*coord/.*$', views.coord, name='coord')
```

Comment:
Well, could you please provide different examples which should be matched here?
I'll suggest something much better.